content/docs/extension-guide/create-extension/setup-local-environment.md

```diff
@@ -72,7 +72,10 @@ We unset any custom lifecycle image that may have been provided to ensure that t
 
 <!-- test:exec -->
 ```bash
-git clone https://github.com/buildpacks/samples.git
+git clone https://github.com/buildpacks/samples.git && \
+  cd samples && \
+  git fetch --all && \
+  git checkout fix/curl
```

Comment:
```suggestion
git clone https://github.com/buildpacks/samples.git
```
When https://github.com/buildpacks/samples/pull/176 is merged we can remove these